src/app.js

```diff
@@ -2,10 +2,14 @@ const express = require('express');
 const path = require('path');
 const handlebars = require('express-handlebars');
 const bodyParser = require('body-parser');
+const compression = require('compression');
+const controllers = require('./controllers/index');
```

Comment:
You don't need to call `index`, nodejs automatically looks for index.js, if it isn't found then it'll produce an error.